discovery-infra/test_infra/helper_classes/infra_env.py

```diff
@@ -120,4 +121,11 @@ def patch_discovery_ignition(self, ignition_info: str) -> str:
         self.api_client.patch_discovery_ignition(infra_env_id=self.id, ignition_info=ignition_info)
 
     def get_details(self) -> models.infra_env.InfraEnv:
-        return self.api_client.get_infra_env(infra_env_id=self.id)
\ No newline at end of file
+        return self.api_client.get_infra_env(infra_env_id=self.id)
+
+    def update_proxy(self, proxy: models.Proxy) -> None:
```

Comment:
Can you also please change `cluster.set_proxy_values` to match the same signature?